src/journal/models.py

```diff
@@ -511,6 +512,25 @@ def description_for_press(self):
                 setting_name='journal_description',
             )
 
+    def setup_default_review_form(self):
+        default_review_form = review_models.ReviewForm.objects.create(
+            journal=self,
+            name='Default Form',
+            intro='Please complete the form below.',
+            thanks='Thank you for completing the review.'
+        )
+
+        main_element = review_models.ReviewFormElement.objects.create(
+            name='Review',
+            kind='textarea',
+            required=True,
+            order=1,
+            width='large-12 columns',
+            help_text=gettext('Please add as much detail as you can.'),
+        )
+
+        default_review_form.elements.add(main_element)
```

Comment:
Should we use `.get_or_create()` to avoid duplicates and having to be overly defensive when calling this?